operations/routingrules.md

```diff
@@ -0,0 +1,380 @@
+# Routing rules
+
+## Get all routing rules
+
+Returns all routing rules.
```

Comment:
It is correct, because it is one of the required parameters. You either provide a CompanyIds and get all routing rules associated to those, or You provide the specific RoutingRuleIds to get information about those right away.